content/en/blog/_posts/2024-XX-XX-websocket-transition.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: blog
+title: 'Streaming Transitions from SPDY to WebSockets'
+date: 2024-XX-XX
```

Comment:
is this on purpose?

Reply:
Updated the date, now that it's been selected.